Assets/Scripts/Content/ContentData.cs

```diff
@@ -482,7 +483,7 @@ void AddContent(string name, Dictionary<string, string> content, string path, st
         {
             PerilData d = new PerilData(name, content);
             // Ignore invalid entry
-            if (d.name.Equals(""))
+            if (d.sectionName.Equals(""))
```

Comment:
sorry about that. The main problem with name is that it sometimes is used as a localizable text (public name) ane others like a code (internal name). But all of them inherits from the GenericData class and I figured out changing name to StringKey and when name is useda as inernal code, use sectionName instead. On the investigator selection dialog it worked fine but in peril looks like there is some problem? 
I thought about spliting name into codeName and publicName but I wasn't sure about where to use one or other.